new_src/main.py

```
@@ -64,5 +68,79 @@ def run_cli():
+    script_path = os.path.join(root_path, script_name)
+    print(script_path)
+    # 1. 스크립트 파일 존재 여부 확인
+    if not os.path.exists(script_path):
+        print(f"오류: 실행할 스크립트 파일 '{script_name}'을 찾을 수 없습니다.")
+        print("스크립트 파일을 이 파이썬 파일과 같은 위치에 저장했는지 확인해주세요.")
+        sys.exit(1)
+        
+    # 2. 실행 권한 확인 (중요!)
+    if not os.access(script_path, os.X_OK):
+        # 권한 부여
+        os.chmod(script_path, 0o755)
```

Ruff (0.14.3)

`os.chmod` setting a permissive mask `0o755` on file or directory

(S103)